modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -1483,4 +1484,8 @@ private void generateFilesMetadata(List<File> files) {
         }
     }
 
+    private String removeTrailingSlash(String value) {
```

Comment:
Maybe we should pass in a boolean onlySlashBecomesEmptyString?
hat we we could use it to preserve the current behavior here for rust jersey-2 etc.